command/check.go

```diff
@@ -127,3 +129,14 @@ func (c *AgentCheckCommand) checkClientHealth(clientStats map[string]string, min
 
 	return HealthPass
 }
+
+func (c *AgentCheckCommand) AutocompleteFlags() complete.Flags {
+	return complete.Flags{
+		"-min-peers":   complete.PredictNothing,
+		"-min-servers": complete.PredictNothing,
```

Comment:
Things that take something after the command should use `PredictAnything` https://godoc.org/github.com/posener/complete#pkg-variables